WTWSMS/common/dynasties/00_dynasties.txt

```diff
@@ -28130,7 +28130,7 @@
 	culture = old_slavic
 }
 1042167 = {
-	name="Atillid" 
+	name="the Hun" 
 	culture = hunnic 
```

Comment:
1. What is it translated from?

2. -but none that ruled the Hunnic Empire. Attila the Hun, and his powerful family, were the epitome of the Huns and the greatest Hunnish dynasty. Their name was and is synonymous with the Hunnic Empire.

3. Attila the Hun's name (and thus "the Hun") can be translated into French and Spanish. Based on the French Wikipedia translation, "Attila des Huns" (and thus House "des Huns") could pass, but I am likely wrong in use of semantics. I am sure a Frenchman can help us on that.

4. Right now, Hunnish characters can be assigned the first name "Attila". Using the illegitimate cadet dynasty naming conventions, if that Attila is illegitimate and has kids, he can create his own "Attilid" dynasty.

5. The closest thing I could find to a known surname is "**_Attilus_**". Attila the Hun and his family were under House the Hun for years, the change was unneeded and detracting of the dynastic prestige. He deserves an unique dynasty name, not a slapped-together generic dynasty. Look at all the AARs using it. "The Hun" is more accurate than Attilid because, once again, "the Hun" and "Scourge of God" (Flagellum Deo) was a contemporary nickname used in that time period. We could use the dynasty "Flagellum Deo" or "Attilus" as a comrpmise. Both of those make more sense than "Attilid", which is total conjecture. "The Hun" has some basis in history, "Attilid" has nothing but your own imagination.